### PR TITLE
send flutter.error analytics

### PR DESCRIPTION
**Pull request by devoncarew**
- send flutter.error analytics
- create and normalize an error code based on he Flutter.Error summary text
- some unrelated formatting in the logging package

### CODE REVIEW

File: src/io/flutter/logging/FlutterErrorHelper.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    return null;
+  }
+
+  @VisibleForTesting
+  public static String getAnalyticsId(String errorSummary) {
+    // "A RenderFlex overflowed by 1183 pixels on the right."
+
+    // normalize to lower case
```
Comment (devoncarew) [inline src/io/flutter/logging/FlutterErrorHelper.java]: @InMatrix @jacob314 please review the normalization we're doing below

Comment (jacob314) [reply src/io/flutter/logging/FlutterErrorHelper.java]: Lets add a TODO to remove the normalization. With a future change adding a kernel transformer to Flutter for the error description objects, this error summary will start showing up as
`'A $runtimeType overflowed by $overflowText.'`

**Comment by jacob314** [comment]
lgtm

File: src/io/flutter/logging/FlutterErrorHelper.java (hunk 1 above)
Comment (devoncarew) [reply src/io/flutter/logging/FlutterErrorHelper.java]: 👍 done

**Pull request merged**